bundle/src/main/java/com/adobe/acs/commons/httpcache/config/HttpCacheConfig.java

```diff
@@ -77,6 +79,15 @@
      */
     List<Pattern> getJCRInvalidationPathPatterns();
 
+    /**
+     * Get a list of headers (as regex pattern) that should NOT be put in the cached response, to be served to the output.
+     * This is useful for example with systems that put a login cookie in each response.
+     * @return
+     */
+    default List<Pattern> getExcludedResponseHeaderPatterns() {
+        return emptyList();
```

Comment:
Generally better to qualify than making static import; this makes me go look at where it's coming from (ie. did you implement your own for some reason?) I tend to use static imports for constants or in Unit tests where its very obvious what is not provided by custom code (ie. assertXXX(..), mock(..), etc.)
```suggestion
        return Collections.emptyList();
```